compliant-sets/README.md

```diff
@@ -0,0 +1,9 @@
+# OpenAPI Overlay Compliant Sets
```

Comment:
```suggestion
# OpenAPI Overlay Compliance Tests
```
Or similar, and rename the top-level folder accordingly

Reply:
I think "Tests" rather overstates what these are: more like working examples.

Reply:
I would expect that a compliant implementation produces equivalent output for the two inputs, and that implementors are well advised including these triplets in their end-to-end tests.